marlowe-test/stress-testing.md

```diff
@@ -0,0 +1,111 @@
+# Stress Testing for Marlowe Runtime
+
+This document summarizes the several techniques and tools that can be used for stress testing a [Marlowe Runtime deployment](../marlowe-runtime/).
+
+
+# Fetching History of All Marlowe Contracts
+
+The `marlowe-finder` tool discovers and fetches the history of all Marlowe contracts of registered Marlowe versions. It stresses the synchronization API within the `marlowe-sync` backend service. See the [`marlowe-finder` documentation](../marlowe-apps/Finder.md) for details.
+
+If Marlowe Runtime is available on the default host and ports, this application can be run at the command line with no arguments:
+```bash
+marlowe-finder
+```
+It will write voluminous output the standard error. That output can be filtered as follows, so it just lists and watches the currently active contracts:
+```bash
+marlowe-finder |& jq 'select(.FinderProcess.fields.action == "wait") | .FinderProcess.fields | del(.action)'
+```
+
+The Cardano public  `preview`  test network currently contains well over 100,000 Marlowe transactions, which makes it an ideal network on which to run `marlowe-finder` . A successful stress test is to simply run `marlowe-finder` and observe whether any backend services report errors in the logs or crash.
+
+
+## Running Multiple Marlowe Contract Simultaneously
```

Comment:
Suggest making 'Contract' plural in this heading

Reply:
Fixed in 908a7e461d7bc492d89665f3ab46ef743ea02c76